Assets/GothicVR/Scripts/Player/Running/MovementTypeController.cs

```diff
@@ -0,0 +1,203 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.XR;
+using UnityEngine.XR.Interaction.Toolkit;
+using GVR.Manager;
+
+namespace GVR.Player
+{
+    public class MovementTypeController : MonoBehaviour
+    {
+        //No UI icon for now 
+        //public Canvas runningCanvas;
+        public GameObject leftHand;
+        public GameObject rightHand;
+        public CharacterController character;
+
+        public static float walking_speed;
+        public static float running_speed;
+        private bool _runClickHandled = false;
+
+        private Vector3 previousPosLeft;
+        private Vector3 previousPosRight;
+
+        private const float MIN_VELOCITY_ARM_SWING_ACTIVATE = 0.02f;
```

Comment:
maybe increase the value a bit, i just need to rotate my hands and it triggers the arm swinging